components/Header.vue

```diff
@@ -19,3 +22,12 @@ export default {
   })
 }
 </script>
+
+<style scoped>
```

Comment:
a v-img might be better here, like the one used in the [vuetify doc](https://github.com/vuetifyjs/vuetify/blob/d06d8c99d2a5156492e36e01efd63bb3232a922b/packages/docs/src/components/core/Toolbar.vue#L21-L27)